content/en/apps/tutorials/local-setup.md

```diff
@@ -94,12 +97,23 @@ This CHT instance is empty and has no data in it. While you're free to explore a
 
 ### 2. Install cht-conf
 
-Using npm and python on your terminal, install cht-conf and pyxform globally using the following commands:
-
+Using npm on your terminal, install cht-conf globally using the command below. Confirm if you have previously installed cht-conf.
+If you have, skip this step.
 ```shell
 npm install -g cht-conf
+```
+Using python on your terminal, install pyxform globally using the command below. 
+```shell
 sudo python -m pip install git+https://github.com/medic/pyxform.git@medic-conf-1.17#egg=pyxform-medic
 ```
+Incase you encounter issues while installing pyxform using the command above. Paste the commands below in your terminal:
```

Comment:
I believe that the issue is related to `python` rather than `pyxform`. I would explain that one needs to install python 2 to have the command above running correctly.

Reply:
We have a "Troubleshooting" section on this already, I recommend:

```suggestion
If you encounter the error `npm ERR! gyp ERR verb find Python Python is not set` while installing pyxform and are running macOS, see [this troubleshooting section]({{< relref "contribute/code/core/dev-environment#macos--123" >}}).
```

And then we should remove the code below.

Reply:
This has been added